frontend/src/components/common/progress-details/ProgressDetails.tsx

```diff
@@ -217,8 +236,47 @@ const ProgressDetails = ({
             <td className={'md'}>Recipient does not exist</td>
             <td className={'sm'}>{invalid}</td>
           </tr>
+          {type === ChannelType.Email && (
+            <tr>
+              <td className={cx(styles.status, 'md')}>
+                <i
+                  className={cx(styles.icon, styles.red, 'bx bx-error-circle')}
+                ></i>
+                Unsubscribers
+              </td>
+              {redacted ? (
+                <td className={cx('md', styles.grayText)}>Report redacted</td>
+              ) : (
+                <td
+                  className={'md'}
+                  style={{ cursor: 'pointer' }}
+                  onClick={exportUnsubscribers}
+                >
+                  <span className={cx(styles.linkLike, styles.textUnderline)}>
+                    Unsubscribe list
+                  </span>
+                  <i
+                    className={cx(styles.icon, styles.blue, 'bx bx-download')}
+                    style={{ marginLeft: '5px' }}
+                  ></i>
+                </td>
+              )}
+              <td className={'sm'}>{unsubscribed}</td>
+            </tr>
+          )}
         </tbody>
       </table>
+      <InfoBlock className={styles.notice}>
+        We recommend that you remove unsubscribers from your campaign to avoid
+        being marked as spam.{' '}
```

Comment:
<img width="613" alt="image" src="https://user-images.githubusercontent.com/67887489/186154267-7356839d-7705-453a-a687-b59045ad3e65.png">

I would prefer stronger language here and maybe some bolding.

> You **should remove** unsubscribers from future campaigns to avoid having them marked as spam and impacting your agency's reputation.

Reply:
This is the design

Reply:
haha I know can we change the design @khaleedah

Reply:
Ok –  I understand the need for the phrasing to be stronger but I don't think it needs to sound unnecessarily authoritative and aggressive - could be more turn off to the agencies. 

Could we rephrase it as the one below (action word upfront, added more context to the consequences so that they would be intrinsically motivated)  

<img width="658" alt="Screenshot 2022-08-31 at 11 14 37 AM" src="https://user-images.githubusercontent.com/87360573/187585228-7f970fd4-604d-4afc-bd2c-cbc316f7dec9.png">

> **Remove unsubscribers from your recipient list**, to avoid future campaigns being marked as spam and affecting the reputation of your agency. Learn more

Reply:
sounds good!